.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: DawidWesierski4, PanKaker, Sakoram
+
+---
+
+Hi! Thank you for submitting the bug.
+Please provide more details below:
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Please provide steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Configuration and command-line**
+Please provide the command-line and configuration files of MTL execution.
+
+**Status report**
+Please execute and provide [status report](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/script/status_report.sh).
+
+**Logs**
+Please provide logs from the application
```

Comment:
I think we should specify the level of logs we expect from the user 
The NOTICE logs don't really bring much to the table